packages/@aws-cdk/aws-lambda-python/lib/Dockerfile

```diff
@@ -9,6 +9,8 @@ ARG HTTPS_PROXY
 
 # Upgrade pip (required by cryptography v3.4 and above, which is a dependency of poetry)
 RUN pip install --upgrade pip
-RUN pip install pipenv poetry
+
+# pipenv 2022.4.8 is the last version with Python 3.6 support
+RUN pip install pipenv==2022.4.8 poetry
```

Comment:
We probably should handle this the same way we handle `awscli` with `lambda-layer-awscli`

https://github.com/aws/aws-cdk/blob/10f5ede7ed4b835df6406f1b457e2b1230bef4df/packages/%40aws-cdk/lambda-layer-awscli/layer/Dockerfile#L18-L19